src/services/PodDatabase.ts

```diff
@@ -96,11 +96,16 @@ export default class PodDatabase {
   }
 
   public async addPod (params: AddPodParams) {
+    const existing = this.pods.get(params.id)
+    const uptime = existing && existing.totalUptime ? existing.totalUptime + Number(params.duration) : Number(params.duration)
```

Comment:
this could just be `const uptime = existing && existing.totalUptime || 0 + Number(params.duration)`